md.py

```diff
@@ -5,6 +5,8 @@
 import pathlib
 import re
 import subprocess
+import sys
```

Comment:
In the commit message:
s/then immediately stopping the arary./then immediately creating the array

Reply:
Actually hmmm. I'm investigating the possibility that the need for a delay here has been caused by a bug in a recent patch. I'm also failing an mdadm test all of a sudden.